roles/bind/tasks/create_keys.yml

```diff
@@ -1,15 +1,18 @@
 ---
 - name: create KSK
-  command: dnssec-keygen -K /var/named/keys -f KSK -3 -a RSASHA256 -b 2048 -r /dev/random -n ZONE ha.{{ base_domain }}
+  ansible.builtin.command: |
+    dnssec-keygen -K {{ dns_work_dir }}/keys -f KSK -3 -a RSASHA256 -b 2048  -n ZONE "hb.{{ base_domain }}"
+  creates: "{{ dns_work_dir }}/keys/Khb.{{ base_domain }}.+*"
```

Comment:
Did you test this? It hasn't been used for at least 8 years (the previous change was manually)

Reply:
Yes, that's how I found out that the -r flag is obsolete